src/app/component/eco-news/components/eco-news-detail/eco-news-widget/eco-news-widget.component.spec.ts

```diff
@@ -0,0 +1,59 @@
+import { TranslateModule } from '@ngx-translate/core';
+import { async, ComponentFixture, TestBed, inject } from '@angular/core/testing';
+
+import { EcoNewsWidgetComponent } from './eco-news-widget.component';
+import { RouterTestingModule } from '@angular/router/testing';
+import { NewsListGalleryViewComponent } from '../../news-list/news-list-gallery-view/news-list-gallery-view.component';
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { of } from 'rxjs';
+import { EcoNewsService } from '@eco-news-service/eco-news.service';
```

Comment:
Don't use real service

Reply:
This is my mistake
i am fixing it in another PR